### DISCUSSION TITLE
/environment-variables-hugo-show-branch-information/

### DISCUSSION THREAD

**Discussion by comment-migrator** [estruyf/eliostruyf-comments#554] [Comments]
Use environment variables in Hugo to show branch information
        
[https://www.eliostruyf.com/environment-variables-hugo-show-branch-information/](https://www.eliostruyf.com/environment-variables-hugo-show-branch-information/)

**Comment by anastasiiaxfr** [comment]
Hello. Tell me please How use variables from .env in JS when in project only Hugo.
